website/blog/2025-05-21-podman-ai-lab-openvino.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Enabling OpenVINO Inference in Podman AI Lab
+description: Use Podman AI Lab to run OpenVINO inference workloads locally.
+authors: [jeffmaury]
+tags: [podman-desktop, podman, extension, ai, llm, local, podman-ai-lab, openvino]
+hide_table_of_contents: false
+---
+
+# Enabling OpenVINO Inference in Podman AI Lab
+
+## Introduction to Podman AI Lab
+
+Podman AI Lab is an open-source platform designed to simplify the deployment, management, and experimentation of AI workloads using container technology. It provides a user-friendly interface for running, testing, and scaling AI models locally or in the cloud, leveraging the power and flexibility of Podman containers.
```

Comment:
Should we link the Podman AI Lab page on the docs (https://podman-desktop.io/docs/ai-lab)? For the users that don't know about the extension yet.